README.md

```diff
@@ -24,7 +24,7 @@ If you've created an extension with GDevelop, you can submit it to be shared wit
 3. **Export** your extension to a _.json file_ from GDevelop.
 4. Submit it! You can either:
    - **Easy**: [submit it here](https://github.com/4ian/GDevelop-extensions/issues/new/choose), attaching the _.json file_ (as a zip, because GitHub won't accept json files directly).
-   - **Faster to review**: fork this repository, clone the git, add your .json file in `Extensions` folder. Finally [open a Pull Request](https://github.com/4ian/GDevelop-extensions/compare). Deployment is done automatically when pushed to `main` branch: [![CircleCI](https://circleci.com/gh/GDevelopApp/GDevelop-extensions/tree/main.svg?style=svg)](https://circleci.com/gh/GDevelopApp/GDevelop-extensions/tree/main)
+   - **Faster to review**: fork this repository, clone the git, add your .json file in `extensions/community` folder. Finally [open a Pull Request](https://github.com/4ian/GDevelop-extensions/compare). Deployment is done automatically when pushed to `main` branch: [![CircleCI](https://circleci.com/gh/GDevelopApp/GDevelop-extensions/tree/main.svg?style=svg)](https://circleci.com/gh/GDevelopApp/GDevelop-extensions/tree/main)
```

Comment:
Maybe the readme should also say how to request it being reviewed and added to the other list 🤔

Reply:
Yes it should, though this can be added in another PR because this will be mostly useful for a small number of users doing really good extensions :) At which point we can as a team also take care of moving the extension.